app/views/admin/conference/edit.html.haml

```diff
@@ -18,5 +18,6 @@
         = f.input :timezone, :as => :time_zone, :hint => "The conference time zone"
         = f.input :start_date, :as => :string, :input_html => { :id => "conference-start-datepicker", :readonly => "readonly"  }
         = f.input :end_date, :as => :string, :input_html => { :id => "conference-end-datepicker", :readonly => "readonly"  }
+      = f.inputs :name => "Registrations" do
+        = f.input :registration_limit, :as => :number, :in => 0..9999, :hint => "Limit the number of registrations to the conference (0 no limit)"
```

Comment:
Can you please use the new ruby syntax?
http://breakthebit.org/post/8453341914/ruby-19-and-the-new-hash-syntax